anyway/views/news_flash/api.py

```diff
@@ -175,7 +175,7 @@ def gen_news_flash_query(
         and_(
             NewsFlash.accident == True,
             not_(and_(NewsFlash.lat == 0, NewsFlash.lon == 0)),
-            not_(and_(NewsFlash.lat == None, NewsFlash.lon == None)),
+            not_(and_(NewsFlash.lat is None, NewsFlash.lon is None)),
```

Comment:
Notice it's not a pythonic operation but SqlAlchemy, from reading this message on Stackoverflow(below) i don't believe it will work

https://stackoverflow.com/a/5632224/13771969